src/helpers/figures/charts/combo_chart.ts

```diff
@@ -83,6 +86,12 @@ export class ComboChart extends AbstractChart {
     this.aggregated = definition.aggregated;
     this.dataSetsHaveTitle = definition.dataSetsHaveTitle;
     this.dataSetDesign = definition.dataSets;
+    if (!this.dataSetDesign.some((design) => design?.bar)) {
```

Comment:
I don't think this is correct. If the user manually set all the datasets to "line", we should not force a random dataset to be a bar. We should distinguish between `bar === false` and `bar === undefined`